api/capi/src/nntrainer_util.cpp

```diff
@@ -41,9 +41,6 @@ ml_optimizer_to_nntrainer_type(ml_train_optimizer_type_e type) {
     throw nntrainer::exception::not_supported(
       "[ml_optimmizer_to_nntrainer_type] Not supported type given");
   }
-
-  throw nntrainer::exception::not_supported(
```

Comment:
I wonder why this is deleted? We know this is unreachable section but this is to prevent no return from future code changes

Reply:
And for others

Reply:
Yes this is unreachable code and coverity point this out.
For that reason should I comment this line?

Reply:
IMO, it's better to tell coverity it's a false positive.

Reply:
I classified it as intentional.